pom.xml

```diff
@@ -48,7 +48,7 @@
     <properties>
         <revision>1.27</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.332.1</jenkins.version>
+        <jenkins.version>2.414.3</jenkins.version>
```

Comment:
If you change the jenkins version you need to change the bom from ` <artifactId>bom-2.332.x</artifactId>` to ` <artifactId>bom-2.414.x</artifactId>` and update the version

Reply:
You will also need to adjust the Jenkinsfile and remove building with java 8. This should be done in a separate PR which gets merged first probably.

Reply:
> This should be done in a separate PR which gets merged first probably.

I think it will need to be in this pull request because I'm not confident that the previous parent pom version supports Java 11.

Reply:
I think that 2.401.3 is a better choice in this case based on the [installation statistics of the plugin](https://stats.jenkins.io/pluginversions/cppcheck.html).  Only 50% of the installations of the most recent release are on Jenkins 2.401.3 or newer.  The ["choosing a version" page](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#currently-recommended-versions) offers either 2.401.3 or 2.414.3 as good choices.  I'd choose 2.401.3 so that more users can choose to upgrade to the next release of the plugin